bin/fuel-core/src/cli/run.rs

```diff
@@ -235,8 +236,8 @@ pub struct Command {
     #[cfg(feature = "p2p")]
     pub sync_args: p2p::SyncArgs,
 
-    #[arg(long = "metrics", env)]
-    pub metrics: bool,
+    #[arg(long = "disable-metrics", value_delimiter = ',', help = fuel_core_metrics::config::help_string(), env)]
+    pub metrics: Option<fuel_core_metrics::config::Config>,
```

Comment:
```suggestion
    #[arg(long = "disable-metrics", value_delimiter = ',', default_value="", help = fuel_core_metrics::config::help_string(), env)]
    pub metrics: fuel_core_metrics::config::Config,
```
you'll need to refactor all the occurrences of `maybe_metrics_config` not to use `.map_or` if you accept this suggestion

Reply:
I like this, it'll make the code a bit simpler. Updated in https://github.com/FuelLabs/fuel-core/pull/2310/commits/e0124b6ef81ce75a5b489f0d4602e0e103cf839b